### ISSUE TITLE
apu3 build - binary filename

### ISSUE THREAD

**Issue by macpijan** [pcengines/release_manifests#13]
apu3 build results in file named `apu2_${VERSION}.rom`

**Comment by macpijan** [comment]
@pietrushnic @mek-x I think it it is confusing. I think build script could use some improvements. For example I don't think there is a way to `distclean` apu3 build.

**Comment by pietrushnic** [comment]
@macpijan I agree about confusing part, but what you mean that we can't do `distclean` on apu3 ?

**Comment by macpijan** [comment]
@pietrushnic For example: https://github.com/pcengines/apu2-documentation/blob/master/scripts/build_release_img.sh#L85
To build legacy apu3 we have to pass parameters: `build apu3`
To execute `distclean` we have to pass `build distclean` which will perform clean build for apu2.
Maybe switches would me more flexible and user-friendly than positional parameters

**Comment by pietrushnic** [comment]
@macpijan I believe so. If you can make it more consistent please do the fix.
